modules/social_features/social_follow_taxonomy/modules/social_follow_tag/src/Plugin/ActivityContext/FollowTagActivityContext.php

```diff
@@ -70,51 +79,4 @@ public function getRecipientsWhoFollowTaxonomy(array $related_entity, array $dat
     return $recipients;
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  public function isValidEntity(EntityInterface $entity) {
```

Comment:
Can we completely skip this step? 

Don't we need to tell our activity creator that only valid entities are those who can actually be followed?